Gemfile

```diff
@@ -37,6 +37,8 @@ gem 'rails_12factor', group: :production
 gem 'globalize'
 gem 'globalize-accessors'
 
+gem 'friendly_id'
```

Comment:
> Nich, the solution you propose was the first Martin suggested. We werent able to make it works so we used friendly_id.

Ah interesting, I wonder want prevented it from working.

> I'll look at the alternative way to see if I can make it work.

Sounds good.

> Hey, I want to thank you for the organisation, it was awesome.

You're most welcome! Thank you for coming out. We forming a great a team here and with @sophiedeziel building on @vivrass's awesome efforts over the last 4 1/2 years.